src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,93 @@
-export const Pagination = () => {};
+import { getNumbers } from '../../utils';
+
+interface IPagination {
+  perPage: number;
+  total: number;
+  page: number;
+  setPage: React.Dispatch<React.SetStateAction<number>>;
+  getCurrentItems: () => string[];
+}
+
+export const Pagination = ({ page, setPage, perPage, total, getCurrentItems }: IPagination) => {
+
+  // Número total de páginas
+  const listNumbers = Math.ceil(total / perPage);
```

Comment:
The variable 'listNumbers' is not descriptive enough. Consider renaming it to 'totalPages' to better reflect its purpose, which is to store the total number of pages.